client/client.go

```diff
@@ -111,6 +148,20 @@ func NewClient(config Config, log logger.Logger) (*Client, error) {
 	return client, nil
 }
 
+func (c *Client) initMetrics() {
+	c.metrics.getReqCount = c.stats.NewTaggedStat("keydb_client_req_total", stats.CountType, stats.Tags{"method": "get"})
+	c.metrics.getReqLatency = c.stats.NewTaggedStat("keydb_client_req_latency_seconds", stats.TimerType, stats.Tags{"method": "get"})
+	c.metrics.getReqFailures = c.stats.NewTaggedStat("keydb_client_req_failures_total", stats.CountType, stats.Tags{"method": "get"})
+	c.metrics.getReqRetries = c.stats.NewTaggedStat("keydb_client_req_retries_total", stats.CountType, stats.Tags{"method": "get"})
+	c.metrics.getKeysQueried = c.stats.NewStat("keydb_client_get_keys_queried_total", stats.CountType)
+	c.metrics.getKeysFound = c.stats.NewStat("keydb_client_get_keys_found_total", stats.CountType)
+
+	c.metrics.putReqCount = c.stats.NewTaggedStat("keydb_client_req_count_total", stats.CountType, stats.Tags{"method": "put"})
+	c.metrics.putReqLatency = c.stats.NewTaggedStat("keydb_client_req_latency", stats.TimerType, stats.Tags{"method": "put"})
```

Comment:
This should also be `_seconds` like the one on line 153, right? Also, we might need buckets for these two.